cookbooks/bcpc/recipes/bach_repository_wrapper.rb

```diff
@@ -0,0 +1,11 @@
+node.force_default['bach']['repository']['gem_server'] = node['bcpc']['bootstrap']['server']
```

Comment:
why use `force_default` here? 

`force_default` is supposed to be reserved for internal wrappers so that it has the highest precedence.  having it in `chef-bach` will confuse the precedence in that part.

Reply:
This is wrapping `bach_repository`'s defaults; as such I can not use the `default` precedence can I?

Reply:
using explicit recipe wrapping like having `include_recipe 'bach_repository::the_recipe_being_wrapped'` at the end of this recipe should make that work.